Add docs for new env vars in actions (#725)
packit/packit.dev: docs/configuration/actions.md

## Environment variables set by packit

> Additionally, packit sets a few env vars for all the actions:
> 
> `PACKIT_UPSTREAM_PROJECT_NAME` — set to the `upstream_package_name` value, if any, or empty string  
> `PACKIT_DOWNSTREAM_PROJECT_NAME` — set to the `downstream_package_name` value, if any, or empty string  
> `PACKIT_PROJECT_NAME` — the package name key for the `packages` dictionary in a monorepo project,
> otherwise the `downstream_package_name` value is used, if any, or the `upstream_package_name`  
> 
> and other env vars for specific actions:

Reviewer: ```suggestion
Additionally, packit sets several environment variables for the actions:

`PACKIT_UPSTREAM_PACKAGE_NAME` — set to the `upstream_package_name` value, if any
`PACKIT_DOWNSTREAM_PACKAGE_NAME` — set to the `downstream_package_name` value, if any
`PACKIT_PACKAGE_NAME` — set to the package name key for the `packages` dictionary in a monorepo project, falls back to `downstream_package_name` or, if not set, to `upstream_package_name`

There are also action-specific environment variables:
```

**create-archive**

> `PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)  

Reviewer: ```suggestion
`PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)
```

`PACKIT_PROJECT_NAME_VERSION` — current name and version of the project (coming from `git describe`)

**changelog-entry**

`PACKIT_PROJECT_VERSION` — version to be set in the specfile, set when relevant 
(e.g. when syncing upstream release downstream)

**release synchronization actions** (`propose-downstream` and `pull-from-upstream`)  
*post-upstream-clone*
*pre-sync*
*prepare-files*
*create-patches*

> `PACKIT_DOWNSTREAM_REPO` — absolute path to the downstream repo working dir  
> `PACKIT_UPSTREAM_REPO` — absolute path to the upstream repo working dir

Reviewer: ```suggestion
`PACKIT_UPSTREAM_REPO` — absolute path to cloned upstream git repo
`PACKIT_DOWNSTREAM_REPO` — absolute path to cloned downstream git repo
```
